src/t2iapi/biceps/codedvalue.proto

```diff
@@ -0,0 +1,40 @@
+/*
+This Source Code Form is subject to the terms of the MIT License.
+Copyright (c) 2024 Draegerwerk AG & Co. KGaA.
+
+SPDX-License-Identifier: MIT
+*/
+
+syntax = "proto3";
+
+package t2iapi.biceps;
+
+option java_package = "com.draeger.medical.t2iapi.biceps";
+option java_outer_classname = "CodedValueProto";
+
+import "t2iapi/biceps/localizedtext.proto";
+import "t2iapi/biceps/codeidentifier.proto";
+import "google/protobuf/wrappers.proto";
+import "t2iapi/biceps/symboliccodename.proto";
+
+/*
+Represents pm:CodedValue (defined in IEEE Std 11073-10207-2017).
+*/
+message CodedValueMsg {
+  repeated LocalizedTextMsg coding_system_name = 1;
+  repeated LocalizedTextMsg concept_description = 2;
+  /*
+  Represents pm:Translation (defined in IEEE Std 11073-10207-2017).
+  */
+  message TranslationMsg {
+    CodeIdentifierMsg code_attr = 3;
```

Comment:
no need for this type indirection, it's a string

Reply:
done